src/pages/members_page.js

```diff
@@ -26,11 +26,11 @@ class Members extends React.Component {
   render() {
     return (
 
-      <Grid fluid>
-        <Row center="xs">
+      <Grid fluid style={{ 'max-width': '1400px' }}>
+        <Row center="lg">
         {this.state.members.map((member) => {
           return (
-            <Col xs={12} sm={6} md={4}>
+            <Col xs={2.5} sm={2.5} md={2.5}>
```

Comment:
can we use sass to constantize these numbers at some point? Not a big deal, but it might help if we ever end up scaling the dashboard

Reply:
That's a fair point. I'm just not sure how I would use Sass outside of the stylesheet file. Maybe we could walk through it together in person.